src/components/SectionCitations.js

```diff
@@ -99,7 +99,7 @@ const SectionCitations = () => {
                     : citation.websiteTitle}
                 </Typography>
                 <Typography>
-                  {moment(citation.date).strftime("%B %e, %Y")}
+                  <em>{moment(citation.date).strftime("%B %e, %Y")}</em>
```

Comment:
This is a solid option for adding italics! However, for best practices, it's best to add a class to italicize the text. This is because different browsers read the `em` tag differently, so it might not show italics in every instance. I'd recommend adding a `className={classes.em}` to the `Typography` element, and then giving it the rule `fontStyle: "italic"`

Reply:
thank youuu!